tests/foreman/ui/test_gpgkey.py

```diff
@@ -556,48 +555,6 @@ def test_positive_consume_content_using_repos_and_different_keys(self):
         @CaseLevel: System
         """
 
-    @stubbed()
-    @run_only_on('sat')
-    @tier1
-    def test_positive_list(self):
-        """Create gpg key and list it
-
-        @id: 9963f533-47aa-49a9-b251-3d81fc07e732
-
-        @assert: gpg key is displayed/listed
-
-        @caseautomation: notautomated
-
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    @tier1
-    def test_positive_search(self):
-        """Create gpg key and search/find it
-
-        @id: d55b8034-7f55-45cc-8ee0-43db0534e586
-
-        @assert: gpg key can be found
-
-        @caseautomation: notautomated
-
-        """
-
-    @stubbed()
-    @run_only_on('sat')
-    @tier1
-    def test_positive_info(self):
-        """Create single gpg key and get its info
-
-        @id: c8b75db1-9394-4a99-9d91-0d388aacfd1a
-
-        @assert: specific information for gpg key matches the creation values
```

Comment:
No questions re 2 others tests, but do we have something similar to this one automated? I mean fetching field values of created gpg key?